src/models/type.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Type model
+ */
+
+import mongoose from 'mongoose';
+
+const TypeSchema = new mongoose.Schema(
```

Comment:
Should be more specific - NotificationTypeSchema instead of TypeSchema

Reply:
done

Reply:
done